tests/generic_tests.rs

```diff
@@ -85,3 +90,31 @@ fn test_mixed_conversions_round_trip() {
     let u_again: SavedUser = convert_from(au);
     assert_eq!(u_again, before)
 }
+
+#[test]
+fn test_pluck() {
```

Comment:
I'm not sure if this is the right place to write a test. I added it as a usage example, but I'll delete it if it's not needed.